help-beta/src/styles/main.css

```diff
@@ -3,3 +3,37 @@
     pointer-events: none;
     cursor: default;
 }
+
+.sl-markdown-content {
+    img {
+        vertical-align: top;
+        box-shadow: 0 0 4px hsl(0deg 0% 0% / 5%);
+        border: 1px solid hsl(0deg 0% 87%);
+        border-radius: 4px;
+        margin-top: 0;
```

Comment:
We need this margin because of this rule applied by starlight:

```
.sl-markdown-content
	:not(a, strong, em, del, span, input, code)
	+ :not(a, strong, em, del, span, input, code, :where(.not-content *)) {
    margin-top: 1rem;
}
```